src/app.py

```diff
@@ -222,6 +223,20 @@ def scitokens():
         return Response("Server error getting scitokens config, please contact help@opensciencegrid.org", status=503)
 
 
+@app.route("/oasis-managers/json")
+def oasis_managers():
+    if not _get_authorized():
+        return Response("Not authorized", status=400)
```

Comment:
```suggestion
        return Response("Not authorized", status=403)
```
Shouldn't it be 403 Forbidden?

Reply:
Eh, sure!